src/test/kotlin/com/google/androidstudiopoet/ModulesWriterTest.kt

```diff
@@ -1,8 +1,27 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
 package com.google.androidstudiopoet
 
+<<<<<<< HEAD
```

Comment:
Please merge here

Reply:
Nice catch! Fixed it. Also added "test" task for TravisCI to prevent it from happening ever again.